packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -79,7 +79,8 @@ governing permissions and limitations under the License.
   .spectrum-Tag-removeButton {
     grid-area: action;
     height: calc(var(--spectrum-tag-height) - (2 * var(--spectrum-tag-border-size)));
-    width: var(--spectrum-global-dimension-size-300);
+    width: unset;
+    aspect-ratio: 1;
```

Comment:
the button becomes a square which will change the width of the button by about 2px. I figured this was acceptable and made more sense, it will change the chromatic, that's expected
We can addin a margin of 1px to either side of the button if we think it's too small now